### PR TITLE
Mayflower V10:  global css

### PR DESCRIPTION
**Pull request by smurrayatwork**
Major Changes:

-  The functionality of `packages/assets/scss/00-base/_global.scss` has changed. Now, this file contains optional miscellaneous styles a site can include once for setting up layout/fonts/normalization.
- The file `packages/assets/scss/00-base/configure.scss` preserves the previous behavior of `packages/assets/scss/00-base/_global.scss`. All files previously pointing to `_global.scss` have been updated to point at `_configure.scss`.
- Mayflower Assets has added functionality to generate `.css` files based on `.scss` files placed under `packages/assets/build`. A minified version of each `.scss` file is also provided as `[scss-filename].min.css`.
- The `_index.scss` files used by Header, HeaderSlim, Footer, and FooterSlim's storybook stories have been moved to `packages/assets/build`.

I also updated Storybook to the latest minor version in this PR.

### CODE REVIEW

**Comment by clairesunstudio** [comment]
Can you git ignore the compiled css files and include the build step in `rush build` in circleCI as a prepublish step?

File: packages/assets/build/global.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@forward "normalize-scss/sass/normalize";
+
+@forward "z-index";
```
Comment (clairesunstudio) [inline packages/assets/build/global.scss:3]: z-index is in configure.scss

Comment (clairesunstudio) [inline packages/assets/build/global.scss:3]:
```suggestion
```

**Comment by clairesunstudio** [comment]
can you update `packages/site` to use `configure.scss`. 
Screen Shot 2020-10-26 at 2 56 58 PM

File: packages/assets/build/_header-search.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      align-items: center;
+    }
+
+    label {
+      position: absolute !important;
+    }
+  }
+}
+
+@include meta.load-css("02-molecules/header-search");
```
Comment (clairesunstudio) [inline packages/assets/build/_header-search.scss:78]: why is this scss loaded at the bottom?

Comment (smurrayatwork) [reply packages/assets/build/_header-search.scss:78]: CSS overrides needed it to be placed there.

**Pull request closed without merging**